### ISSUE TITLE
Debug UI broken in 0.14.0 

### ISSUE THREAD

**Issue by ArnSch** [clojure-liberator/liberator#253]
On trying to debug a request with the wrap-trace middleware in v0.14.0, the decision graph in the UI is broken.

It seems to be an issue with the javascript. I'm getting

```
Uncaught TypeError: Cannot read property 'setAttribute' of null
```

on this line of code
```
svg.getElementById("initializecontext").setAttribute("class", svg.getElementById("initializecontext").getAttribute("class") + " hl-true");
```

It seems that `svg.getElementById("initializecontext")` is null.

**Comment by ordnungswidrig** [comment]
Can you try this with 0.13?

**Comment by ArnSch** [comment]
@ordnungswidrig it works well in 0.13. Any idea about when it will be fixed in 0.14.0 ?

**Comment by ordnungswidrig** [comment]
I need to investigate first. I'm not aware of any change on 0.14 that would obviously break this.
